ts-client/src/dlmm/index.ts

```diff
@@ -4342,15 +4336,16 @@ export class DLMM {
       getOrCreateATAInstruction(
         this.program.provider.connection,
         this.tokenX.publicKey,
-        owner
+        payer,
```

Comment:
This should be `operator` because operator adding liquidity on behalf of position owner. The 4th parameter will be `payer`